app/src/main/java/de/triology/cas/services/dogu/CesDoguServiceFactory.java

```diff
@@ -12,25 +12,47 @@
     public RegexRegisteredService createCASService(long id, String fqdn){
         RegexRegisteredService casService = new RegexRegisteredService();
         casService.setId(id);
-        casService.setServiceId("https://" + fqdn + "/cas/.*");
+        casService.setServiceId("https://" + generateServiceIdFqdnRegex(fqdn) + "/cas/.*");
```

Comment:
You tried something like: https://stackoverflow.com/a/34219588 ? Sounds way easier for me